dev-python/stripe/stripe-2.10.1.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 python3_{4,5,6,7} pypy{,3} )
+inherit distutils-r1
+
+DESCRIPTION="Stripe python bindings"
+HOMEPAGE="https://github.com/stripe/stripe-python"
+SRC_URI="mirror://pypi/s/${PN}/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+# Tests communicate with an HTTP server
+# see https://github.com/stripe/stripe-python/blob/v2.10.1/README.md#development
```

Comment:
Why not try to package it, and run it in `python_test`?

Reply:
I thought tests that communicate over the network are discouraged, because they do not work with `FEATURES="network-sandbox"` which is suggested for [package testing](https://wiki.gentoo.org/index.php?title=Package_testing&oldid=719658).
Or did I get it wrong? I honestly never activated `network-sandbox`.

Reply:
Tests that communicate with *outside* network, yes. But you can start the server in test phase, and then it will be local to the emerge process.

Reply:
Look at pymongo for a crazy example. Though hopefully yours wouldn't have to be that complex.

Reply:
Thanks a lot! Feel free to ping me if you hit any trouble. Given it's golang, @gyakovlev may be of more help though.

Reply:
Should I file a separate Bug and PR for stripe-mock since it's a new package?